app/src/main/java/ch/epfl/sweng/eventmanager/repository/data/Event.java

```diff
@@ -0,0 +1,60 @@
+package ch.epfl.sweng.eventmanager.repository.data;
+
+import android.graphics.Bitmap;
+
+/**
+ * This class holds the basic elements about an organized event.<br>
```

Comment:
An HTML tag crept into this comment!

Reply:
Indeed, JavaDoc can hold HTML tags, especially to force linebreaks